pkg/server/plugin/nodeattestor/k8spsat/psat.go

```diff
@@ -226,6 +226,10 @@ func (p *AttestorPlugin) Configure(_ context.Context, req *configv1.ConfigureReq
 		return nil, status.Error(codes.InvalidArgument, "core configuration missing trust domain")
 	}
 
+	if len(hclConfig.Clusters) < 1 {
```

Comment:
This warning should ideally be moved to down to line 274 just before the call to p.setConfig. The configure call can still fail at this point (i.e., line 240) leaving the configuration unchanged. Issuing the warning before the config is actually changed could be misleading.